release_notes.md

```diff
@@ -2,28 +2,21 @@
 
 ## Playbook Enhancements
 
-- Added a new playbook **Reset Scenario - Get Correlated Records** that fetches all the correlated records of the demo record created for the simulation
-
 - Following playbooks have been modified
 
   - **Run Selected Scenario** playbook modifications:
-
-    - The loop execution mode of the *Create Alert* step is updated from `Sequential` to `Parallel`
-
-    - A `Simulation` tag is added to the playbook
-
-    - The **Create Scenario** step is renamed to **Create Alerts** 
+    - A `SystemWaitForCompletion` tag is added to the playbook
 
   - **Run Scenario** playbook modifications:
-
-    - The loop execution mode of the *Create Alert* step is updated from `Sequential` to `Parallel`
+    - A `SystemWaitForCompletion` tag is added to the playbook
 
   - **Run Scenario - Create Alerts** playbook modifications:
+    - Updated the logic that displays the newly created demo record IRIs in scenario record's "Created Records ID" field. 
```

Comment:
Can we mentioned as "fixed inconsistencies in demo record IRIs in Scenario module"

Reply:
Made changes, please check